src/components/structures/FilePanel.tsx

```diff
@@ -16,9 +16,12 @@ limitations under the License.
 */
 
 import React from 'react';
-import PropTypes from 'prop-types';
 
 import {Filter} from 'matrix-js-sdk/src/filter';
+import { EventTimelineSet } from "matrix-js-sdk/src/models/event-timeline-set";
+import { MatrixEvent, Room } from 'matrix-js-sdk/src';
+import { TimelineWindow } from 'matrix-js-sdk/src/timeline-window';
+
 import * as sdk from '../../index';
```

Comment:
would be good to kill this off during this conversion imo

Reply:
As far as I'm aware it's still needed for tests sometimes?
Under what circumstances is it acceptable to remove?

Reply:
Well this'll be for the `getComponent` calls below, my point was we can probably eradicate them